etcdserver/api/membership/cluster.go

```diff
@@ -262,29 +262,38 @@ func (c *RaftCluster) ValidateConfigurationChange(cc raftpb.ConfChange) error {
 	}
 	switch cc.Type {
 	case raftpb.ConfChangeAddNode, raftpb.ConfChangeAddLearnerNode:
-		if members[id] != nil {
-			return ErrIDExists
-		}
 		urls := make(map[string]bool)
 		for _, m := range members {
 			for _, u := range m.PeerURLs {
 				urls[u] = true
 			}
 		}
-		m := new(Member)
+
+		m := new(ConfigChangeContext)
```

Comment:
Forgot to mention that we probably want a new variable name here. I think `m` was originally used to represent member. which is no longer the case here.

Reply:
ok

Reply:
fixed

Reply:
ok